ios/jailed/Makefile

```diff
@@ -0,0 +1,15 @@
+# Don't modify these three variables
+DEBUG=0
+TARGET_CODESIGN=
+override _LOCAL_USE_SUBSTRATE=
```

Comment:
I think you’ll need to be making a copy so that it can be discerned from the standard `tweak` (e.g. `$(TWEAK_JAILED_NAME)`, `internal-tweak-jailed-all_`), and so you can modify bits specific to jailbreak tweaks (e.g. the install path; the logic for the filter plist).